helpers/aws/functions.js

```diff
@@ -217,14 +218,19 @@ function crossAccountPrincipal(principal, accountId) {
         awsPrincipals = [awsPrincipals];
     }
 
+    var principals = [];
+    var response = false;
+
     for (var a in awsPrincipals) {
-        if (/^arn:aws:iam::[0-9]{12}.*/.test(awsPrincipals[a]) &&
+        if (/^arn:aws:iam|sts::[0-9]{12}.*/.test(awsPrincipals[a]) &&
```

Comment:
double check this regex. I tried it and it wasn't working this way. I added parenthesis and it worked

```suggestion
        if (/^arn:aws:(iam|sts)::[0-9]{12}.*/.test(awsPrincipals[a]) &&
```

Reply:
Yes, you are right. It was working that way as well but this way is better